### PR TITLE
DF-40 Implement concept of notification component

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/webapp/app/blocks/interceptor/api-rate-limit.interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    constructor(private coreStore: Store<CoreState>) {
+    }
+
+    intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+        return next.handle(req).do(() => {
+        }, (err: any) => {
+            if (err instanceof HttpErrorResponse && err.status === 429) {
+                const alert = {
+                    type: 'warning',
+                    message: 'There were to many requests from your ip. Please wait 1 min'
```
Comment (pavax) [inline src/main/webapp/app/blocks/interceptor/api-rate-limit.interceptor.ts]: we need a way to inject the translation service and translate the message

File: src/main/webapp/app/blocks/interceptor/api-rate-limit.interceptor.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    HttpHandler,
+    HttpInterceptor,
+    HttpRequest
+} from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';
+import { CoreState } from '../../shared/state-management/state/core.state';
+import { ShowAlertAction } from '../../shared/state-management/actions/core.actions';
+
+export class ApiRateLimitInterceptor implements HttpInterceptor {
```
Comment (pavax) [inline src/main/webapp/app/blocks/interceptor/api-rate-limit.interceptor.ts]: TooManyRequestsInterceptor

File: src/main/webapp/app/shared/alert/global-alert.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Alert, CoreState, getAlerts } from '../state-management/state/core.state';
+import { Observable } from 'rxjs';
+import { HideAlertAction } from '../state-management/actions/core.actions';
+
+@Component({
+    selector: 'jr2-global-alert',
+    template: `
+        <div *ngIf='(alerts$ | async) as alerts' class="global-alert">
```
Comment (99999989) [inline src/main/webapp/app/shared/alert/global-alert.component.ts]: Use an html template, it's too complex for an inline template I think.

File: src/main/webapp/i18n/translations.csv [hunk 4]
```diff
@@ -657,6 +657,8 @@ global,header.title,Arbeitslosenversicherung ALV,Unemployment insurance,Assuranc
 global,navigation.next,Weiter,Next,Suivant,Seguente
 global,navigation.previous,Zurück,Previous,Précédent,Precedente
 global,navigation.scroll-to-top,Nach oben scrollen,Scroll to top,Défiler vers le haut,Scorrere in cima
+global,alert.error.tooManyRequests.authenticated,"There were to many requests from your account. Please wait 1 minute","There were to many requests from your account. Please wait 1 minute","There were to many requests from your account. Please wait 1 minute","There were to many requests from your account. Please wait 1 minute"
```
Comment (pavax) [inline src/main/webapp/i18n/translations.csv]: add the keys to the jira issue so that we can translate them

**Pull request merged**